CHANGELOG.md

```diff
@@ -0,0 +1,80 @@
+# Changelog
```

Comment:
From my point of view nobody will keep this up to date. We should rather use release notes on tags.